tests/unit/amo/components/TestAddonMoreInfo.js

```diff
@@ -423,6 +423,22 @@ describe(__filename, () => {
     expect(link).toHaveProp('href', `/addon/${addon.slug}/versions/beta`);
   });
 
+  it('does not link to beta versions if they are disabled', () => {
```

Comment:
This spec should describe what's happening at a lower level. I suggest something like "it does not expose beta version properties if disabled"